java/resource-overrides/src/main/java/software/amazon/awscdk/examples/ResourceOverridesStack.java

```diff
@@ -0,0 +1,158 @@
+package software.amazon.awscdk.examples;
+
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
+import software.amazon.awscdk.core.*;
+import software.amazon.awscdk.services.autoscaling.AutoScalingGroup;
+import software.amazon.awscdk.services.autoscaling.AutoScalingGroupProps;
+import software.amazon.awscdk.services.autoscaling.CfnLaunchConfiguration;
+import software.amazon.awscdk.services.ec2.*;
+import software.amazon.awscdk.services.s3.Bucket;
+import software.amazon.awscdk.services.s3.BucketEncryption;
+import software.amazon.awscdk.services.s3.BucketProps;
+import software.amazon.awscdk.services.s3.CfnBucket;
+
+import java.util.Collections;
+
+/**
+ * This is an example of how to override properties of underlying CloudFormation resource of
+ * high-level CDK construct.
+ *
+ * Note: this is just a reference code to show examples of how to use L1 resources.
+ * Running `cdk deploy` on this app will fail, however you can still run `cdk synth` and explore
+ * CloudFormation template that gets generated.
+ */
+class ResourceOverridesStack extends Stack {
+    public ResourceOverridesStack(final Construct parent, final String name) {
+        super(parent, name);
+
+        Bucket otherBucket = new Bucket(this, "Other");
```

Comment:
This should be
``` java
Bucket otherBucket = Bucket.Builder.create(this, "Other");
```
Same goes for the other resource inits

Reply:
Nice, good to know!

I replaced all the `new` with builders, with one exception, I left `new AmazonLinuxImage()` instead of replacing it with `AmazonLinuxImage.Builder.create().build`, that's okay?